### PR TITLE
Added Longest Palindromic Substring in C++ Article

### PR DESCRIPTION
**Pull request by hemabhravee**

### CODE REVIEW

**Comment by hemabhravee** [comment]
If you find the article satisfactory, please add the hacktoberfest label to this PR before merging

**Review by jrg94 — CHANGES_REQUESTED** [review]
Looks great! Have you submitted this over in the main repo yet? Also, make sure to add yourself to the authors file.

**Comment by hemabhravee** [comment]
Do I have to submit this in some other repo as well? If so, please tell me where.

**Comment by jrg94** [comment]
It would go here: https://github.com/TheRenegadeCoder/sample-programs

That way, it can get tested and you can document the results here.

**Comment by hemabhravee** [comment]
Got it! Thanks a lot.

File: projects/longest-palindrome-substring/_posts/2020-10-3-c-plus-plus.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
+---
+title: Longest Palindromic Substring in Every Language
```
Comment (jrg94) [inline projects/longest-palindrome-substring/_posts/2020-10-3-c-plus-plus.md:2]:
```suggestion
title: Longest Palindromic Substring in C++
```

File: projects/longest-palindrome-substring/_posts/2020-10-3-c-plus-plus.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+layout: default
+date: 2019-10-08
+last-modified: 2020-05-02
+featured-image:
+tags: [longest-palindrome-substring]
+authors: 
+  - Sayantan Paul
+  - Hemabh Ravee
```
Comment (jrg94) [inline projects/longest-palindrome-substring/_posts/2020-10-3-c-plus-plus.md:10]:
```suggestion
  - hemabhravee
```

**Review by jrg94 — CHANGES_REQUESTED** [review]
This is awesome, but I don't currently see a working solution to this here: https://github.com/TheRenegadeCoder/sample-programs/tree/master/archive/c/c-plus-plus

I assume you have a pull request that this is referencing, but I don't believe we have testing implemented yet. It's not a big deal, but we'll have to work through the code-side first.

File: projects/longest-palindrome-substring/_posts/2020-10-3-c-plus-plus.md [hunk 3]
```diff
@@ -0,0 +1,261 @@
+---
+title: Longest Palindromic Substring in Every Language
+layout: default
+date: 2019-10-08
+last-modified: 2020-05-02
+featured-image:
+tags: [longest-palindrome-substring]
+authors: 
+  - Sayantan Paul
```
Comment (jrg94) [inline projects/longest-palindrome-substring/_posts/2020-10-3-c-plus-plus.md:9]:
```suggestion

```

**Pull request closed without merging**